src/index.js

```diff
@@ -1,7 +1,160 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from "react-dom";
+import React from "react";
+import L from "leaflet";
+import borderData from "./border.js";
+import countyData from "./vtcountyborder";
+import leafletPip from "@mapbox/leaflet-pip";
 
-ReactDOM.render(
-  <h1>Hello World!</h1>,
-  document.getElementById('root')
-)
\ No newline at end of file
+
+const mapStyle = {
+  width: "100%",
+  height: "400px",
+  pointerEvents: "none"
+};
+
+function getRandomArbitrary(min, max) {
+  return Math.random() * (max - min) + min;
+}
+let vtMinLng=-73.42494466485307;
+let vtMaxLng = -71.51022535353107;
+
+let vtMinLat=42.730315121762715;
+let vtMaxLat = 45.00706691759828
+
+let borderData2 = L.geoJSON(borderData);
+function genmerateRandStart(minLat, maxLat, minLng, maxLng){
+
+let latRand = getRandomArbitrary(minLat, maxLat);
+let lngRand = getRandomArbitrary(minLng, maxLng);
+
+const layerLength = leafletPip.pointInLayer(
+  [lngRand, latRand],
+  borderData2
+).length;
+
+if(layerLength){
+return { lat: latRand, lng: lngRand }
+}else{
+  genmerateRandStart(vtMinLat, vtMaxLat, vtMinLng, vtMaxLng)
+}
+
+}
+
+let randStart = genmerateRandStart(vtMinLat, vtMaxLat, vtMinLng, vtMaxLng);
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handler = this.handler.bind(this);
+  }
+  state = {
+    mapPosition: randStart,
+    corectCounty: "county",
+    points: 10
+  };
+
+  
+
+  handler = (direction, points) => {
+  
+    return this.setState({
```

Comment:
You don't need to return this.setState to change the state of the Map component. This.setState will always modify the component's state whenever it is called.